homeassistant/components/zwave_js/trigger.py

```diff
@@ -0,0 +1,48 @@
+"""Z-Wave JS trigger dispatcher."""
+from __future__ import annotations
+
+import importlib
+from types import ModuleType
+from typing import Any, Callable, cast
+
+from homeassistant.const import CONF_PLATFORM
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.typing import ConfigType
+
+
+def _get_trigger_platform(config: ConfigType) -> ModuleType:
+    """Return trigger platform."""
+    return importlib.import_module(
+        f"..triggers.{config[CONF_PLATFORM].split('.')[1]}", __name__
```

Comment:
This is dangerous, you're throwing user generated data into an import function

Reply:
Why not just have a couple of static classes and decorate each with their type using a [`Registry`](https://github.com/home-assistant/core/blob/dev/homeassistant/util/decorator.py#L8)

Reply:
Or even simpler:

```python
from .triggers import value_updated

TRIGGERS = {
  "value_updated": value_updated
}
```